packages/lending/src/protocols/aave-v2/lending-protocol.test.ts

```diff
@@ -0,0 +1,553 @@
+import { LendingProtocol } from './lending-protocol';
+import * as common from '@protocolink/common';
+import { expect } from 'chai';
+
+describe('Test Aave V2 LendingProtocol', function () {
+  context('Test getPortfolio', function () {
+    const testCases = [
+      {
+        chainId: common.ChainId.mainnet,
+        account: '0xc94680947CF2114ec8eE43725898EAA7269a98c5',
+        blockTag: 18797586,
```

Comment:
Does hardhat create a new fork from the latest block and switch to `blockTag`? Or does hardhat create a fork directly from `blockTag`?